src/components/RevokeContainer/RevokeContainer.tsx

```diff
@@ -0,0 +1,125 @@
+import { FunctionComponent, useState, useEffect } from "react";
+import { Redirect } from "react-router";
+import { useConfigContext } from "../../common/context/config";
+import { RevokeDocumentDropZone } from "./RevokeDocumentDropZone/RevokeDocumentDropZone";
+import { RevokeDocumentTileArea } from "./RevokeDocumentTileArea/RevokeDocumentTileArea";
+import { utils } from "@govtechsg/open-attestation";
+import { RevokeConfirmationModal } from "./RevokeConfirmationModal";
+import { RevokedScreen } from "./RevokedScreen/RevokedScreen";
+import { verificationBuilder, isValid, VerificationFragment, openAttestationVerifiers } from "@govtechsg/oa-verify";
+import { DocumentUploadState } from "../../constants/DocumentUploadState";
+
+const getNotRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment[] =>
+  fragments.filter(
+    (status) => status.name !== "OpenAttestationEthereumDocumentStoreRevoked" && status.status !== "SKIPPED"
+  );
```

Comment:
why?

Reply:
i was following how tt handling the validation of the document

Reply:
will update accordingly